pynxtools_stm/stm_file_parser.py

```diff
@@ -404,6 +419,80 @@ def from_sxm_file_into_template(self, template, config_dict, eln_data_dict):
         # The following function can be used later it link come true in application def.
         # link_implementation(template, nxdl_key_to_modified_key)
         link_seperation_from_hard_code(template, nxdl_key_to_modified_key)
+        self.set_default_values(template)
+
+    @staticmethod
+    def fill_temp_with_required_metadata(template, data_dict, config_dict):
```

Comment:
I am not sure why you have this method seperated from the rest of the metadata. Is it just to hardcode some stuff? In that case, it should be renamed.

Reply:
This methods independent of the object variables. But still bundled with STM file from Nanonis vendors.